src/commands/projects/list.ts

```diff
@@ -38,55 +54,109 @@ Use this command to list projects that are part of a particular organization.`
       id: {header: 'ProjectId'},
       displayName: {header: 'Name'},
       languageTopic: {header: 'LanguageTopic'},
-      channels: {
-        header: 'Channels',
-        get: asChannelsList,
-      },
+      ...(this.options['exclude-channels'] ? {} : this.channelsColumn),
       datapacks: {
         header: 'DataPacks',
         get: asDataPackslist,
       },
+      ...(this.options['include-features'] ? this.featuresColumn : {}),
       createTime: {header: 'CreateTime'},
       updateTime: {header: 'UpdateTime'},
     }
   }
 
+  get channelsColumn() {
+    return {
+      channels: {
+        header: 'Channels',
+        get: asChannelsList,
+      },
+    }
+  }
+
+  get featuresColumn() {
+    return {
+      features: {
+        header: 'Features',
+        get: ({enginePackFeatures}: any) => enginePackFeatures.join(','),
+      },
+    }
+  }
+
   get domainOptions(): DomainOption[] {
     debug('get domainOptions()')
     return ['organization']
   }
 
   async buildRequestParameters(options: Partial<flags.Output>): Promise<ProjectsListParams> {
     debug('buildRequestParameters()')
-    const {organization: orgId} = options
+    const {
+      'exclude-channels': excludeChannels,
+      'include-features': includeFeatures,
+      limit = defaultLimit,
+      offset,
+      organization: orgId,
+      sort: sortBy,
+      'with-name': filter,
+    } = options
 
-    return {orgId}
+    return {
+      excludeChannels,
+      includeFeatures,
+      ...(typeof filter === 'undefined' ? {} : {filter}),
+      ...(typeof limit === 'undefined' ? {} : {limit}),
+      ...(typeof offset === 'undefined' ? {} : {offset}),
+      ...(typeof orgId === 'undefined' ? {} : {orgId}),
+      ...(typeof sortBy === 'undefined' ? {} : {sortBy}),
+    }
   }
 
   doRequest(client: MixClient, params: ProjectsListParams): Promise<MixResponse> {
     debug('doRequest()')
     return ProjectsAPI.listProjects(client, params)
   }
 
-  outputHumanReadable(transformedData: any) {
+  outputHumanReadable(transformedData: any, options: any) {
     debug('outputHumanReadable()')
+    const {columns, context} = this
+    const count: number = context.get('count')
+    const offset: number = context.get('offset')
+    const totalSize: number = context.get('totalSize')
+    const shouldIncludeFeatures = options['include-features']
+
     if (transformedData.length === 0) {
       this.log('No projects found.')
 
       return
     }
 
-    this.outputCLITable(transformedData, this.columns)
+    const resultInformation = offset + count > 1 ? `${chalk.cyan(offset + 1)}-${chalk.cyan(offset + count)}` : chalk.cyan(offset + count)
```

Comment:
Whether to display a single number or a range depends on the value of `count` only.